webhook_receiver/settings/production.py

```diff
@@ -28,23 +28,18 @@
 # The "make migrate" ("manage.py migrate") functionality needs to
 # honour the DB_MIGRATION_ environment variables.
 DB_OVERRIDES = dict(
-    PASSWORD=env.str('DB_MIGRATION_PASS',  # noqa: F405
-                     default=DATABASES['default']['PASSWORD']),  # noqa: F405
-    ENGINE=env.str('DB_MIGRATION_ENGINE',  # noqa: F405
-                   default=DATABASES['default']['ENGINE']),  # noqa: F405
-    USER=env.str('DB_MIGRATION_USER',  # noqa: F405
-                 default=DATABASES['default']['USER']),  # noqa: F405
-    NAME=env.str('DB_MIGRATION_NAME',  # noqa: F405
-                 default=DATABASES['default']['NAME']),  # noqa: F405
-    HOST=env.str('DB_MIGRATION_HOST',  # noqa: F405
-                 default=DATABASES['default']['HOST']),  # noqa: F405
-    PORT=env.str('DB_MIGRATION_PORT',  # noqa: F405
-                 default=DATABASES['default']['PORT']),  # noqa: F405
-    OPTIONS=env.json('DB_MIGRATION_OPTIONS',  # noqa: F405
-                     default=DATABASES['default']['OPTIONS']),  # noqa: F405
+    PASSWORD=env.str('DB_MIGRATION_PASS', default=None),  # noqa: F405
+    ENGINE=env.str('DB_MIGRATION_ENGINE', default=None),  # noqa: F405
+    USER=env.str('DB_MIGRATION_USER', default=None),  # noqa: F405
+    NAME=env.str('DB_MIGRATION_NAME', default=None),  # noqa: F405
+    HOST=env.str('DB_MIGRATION_HOST', default=None),  # noqa: F405
+    PORT=env.str('DB_MIGRATION_PORT', default=None),  # noqa: F405
+    OPTIONS=env.json('DB_MIGRATION_OPTIONS', default=None),  # noqa: F405
+
 )
 for override, value in DB_OVERRIDES.items():
-    DATABASES['default'][override] = value  # noqa: F405
+    if value:
```

Comment:
This feels a little more elegant than the previous approach: rather than overriding the default unconditionally with the envar, but but using the original value as _the envar's_ default, only override the default if the envar is set. :)

Reply:
This is much better :+1: